ajax/libs/gridder/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "gridder",
+  "description": "A jQuery plugin that displays a thumbnail grid expanding preview similar to the effect seen on Google Images.",
+  "filename": "jquery.gridder.min.js",
```

Comment:
Let's use `"filename": "js/jquery.gridder.min.js",` because `jquery.gridder.min.js` is under `dist/js` and the basePath is `dist` so what the `jquery.gridder.min.js` file under the folder is `js`.
Thank you.